index.js

```diff
@@ -20,8 +20,60 @@ mongoose
     return self.connection.dropDatabase();
   })
   .then(() => {
-    // Run your code here, after you have insured that the connection was made
+    let firstIteration = Recipe.create(data[0])
```

Comment:
don't name your variables firstIteration, twoIteration and so on. It is better to put descriptive names for what will be stored inside the variable.